src/lib/agb_flash/agb_flash_le_512k.c

```diff
@@ -126,6 +126,13 @@ u16 ProgramFlashSector_LE(u16 sectorNum, void *src)
     if (sectorNum > 15)
         return 0x80FF;
 
+#if AGBFLASH_USE_V126
```

Comment:
😢 Think we should keep the abg flash versions separate no? Feel free to dump under a different directory or something?

Reply:
Nope, I think it's better to have one file for all three, so that all changes are apparent at a glance, and potential bugfixes only need to occur once.

We can argue about the condition of the `#if` though, I can totally see there being a better solution. :D

Reply:
Fine :/